modules/stochastic_tools/test/tests/multiapps/batch_commandline_control/gold/master_multiple_out_storage_0002.csv

```diff
@@ -1,4 +1,4 @@
-sample_0
-8.47877
-6.82795
-5.96624
+sample
+8.9708
```

Comment:
do you know why all random numbers are changed?

Reply:
Yes, as you discovered the current implementation calls getSamples once during initialization, then save state, which is wrong. This fixes that problem.

Reply:
👍